march_rqt_gait_generator/gaits/Example/Noice.gait

```diff
@@ -0,0 +1,345 @@
+gait: "Example"
+subgait: "Subgait placeholder"
+version: "Noice"
+description: "Not very good probably"
+duration: 
+  secs: 6
+  nsecs:         0
+actual_setpoints: 
```

Comment:
using points and setpoints is a bit confusing, we could change this to use user_defined_points